docs/example_pipelines/condition.yaml

```diff
@@ -452,7 +452,7 @@ spec:
             f.close()
           - $(inputs.params.operand1)
           - $(inputs.params.operand2)
-          image: python:alpine3.6
+          image: quay.io/hukhan/python:alpine3.6
```

Comment:
The intent of this is good, but I think we should stay away from using @HumairAK 's personal quay repo.  Could we instead use the UBI python image (`registry.access.redhat.com/ubi8/python-38`), which is also used elsewhere in code/configs around here?

Reply:
ff6e701 : applied code review.

Tested manually, and still working as intended:

![Screenshot 2023-09-01 at 21 18 54](https://github.com/opendatahub-io/data-science-pipelines-operator/assets/1699252/8c44e294-09bb-4a7a-8f5c-054a60f13d55)

showing new image as suggested in code review behind the scenes:

![Screenshot 2023-09-01 at 21 20 54](https://github.com/opendatahub-io/data-science-pipelines-operator/assets/1699252/3a64a866-5276-4ea2-bf39-d854e66b5d15)

also in my understanding registry.access.redhat.com does supports unauthenticated access which is paramount being this quickstart upstream/OSS/community -oriented.

Thank you for your guidance @gmfrasca !